tox.ini

```diff
@@ -27,7 +27,7 @@ deps =
     python-memcached
     watchdog
 
-    hypothesis: hypothesis
+    hypothesis: hypothesis=3.41.0
```

Comment:
are you sure this shouldn't be `==`? (assuming it's passed to `pip install` by tox)

Reply:
Hey, I was trying to check why hypothesis fails on 3.6 last working build was in version ```3.41.0``` but this didn't help so I reverted it.